Remotion/Web/Development.WebTesting.IntegrationTests/AnchorControlObjectTest.cs

```diff
@@ -140,9 +140,10 @@ public void TestClick ()
       home = htmlAnchor.Click().Expect<WxePageObject>();
       Assert.That(home.Scope.FindId("TestOutputLabel").Text, Is.EqualTo("MyHtmlAnchor"));
 
-      var htmlAnchorWithJavaScriptLink = home.Anchors().GetByLocalID("MyHtmlAnchorWithJavaScriptLink");
-      home = htmlAnchorWithJavaScriptLink.Click().Expect<WxePageObject>();
-      Assert.That(home.Scope.FindId("TestOutputLabel").Text, Is.EqualTo("MyHtmlAnchorWithJavaScriptLink"));
+      // TODO: does not work with CSP
+      // var htmlAnchorWithJavaScriptLink = home.Anchors().GetByLocalID("MyHtmlAnchorWithJavaScriptLink");
+      // home = htmlAnchorWithJavaScriptLink.Click().Expect<WxePageObject>();
+      // Assert.That(home.Scope.FindId("TestOutputLabel").Text, Is.EqualTo("MyHtmlAnchorWithJavaScriptLink"));
```

Comment:
Do we have a ticket to track this TODO? If we do, we should link it here (see other TODO comments). Otherwise, if we decide its not worth it, I would remove this code entirely.